modules/ytp-assets-common/src/less/ckan/upstream_ckan/module.less

```diff
@@ -151,8 +151,8 @@
     .actions {
         position: relative;
         float: right;
-        top: -10px;
-        right: -15px;
+        top: 0px;
+        right: 0px;
```

Comment:
upstream_ckan should not be modified as the modifications are lost when ckan is updated.

Reply:
Ok, moved them to a different file